addons/account/models/account_move_line.py

```diff
@@ -1154,6 +1154,7 @@ def action_register_payment(self):
             'context': {
                 'active_model': 'account.move.line',
                 'active_ids': self.ids,
+                'force_group_payment': True,
```

Comment:
I agrred with you yesterday but after thinking about it, wouldn't take make **every** group payment batched by default ? The desired behavior for this task is to batch grouped payments done via the Amounts to Settle view only. I fear this would change the behavior everywhere :/

Reply:
Changed my code to match the expected behavior of the task : Only change in automatic group payment is when doing a batched payment from the Amounts to Settle list view. If possible, the payment will be grouped by default. Any other behavior on the grouped payment stayed the same.